src/main/java/com/blazemeter/ciworkflow/CiBuild.java

```diff
@@ -131,7 +134,7 @@ public void waitForFinish(Master master) throws InterruptedException, IOExceptio
             long now = System.currentTimeMillis();
             long diffInSec = (now - start) / 1000;
             if (now - lastPrint > bzmMinute) {
-                notifier.notifyInfo("BlazeMeter test# , masterId # " + master.getId() + " running from " + start + " - for " + diffInSec + " seconds");
+                notifier.notifyInfo("BlazeMeter test # " + testId + ", masterId # " + master.getId() + " running from " + startTime.getTime().toString() + " - for " + diffInSec + " seconds");
```

Comment:
* Could you give an example of this format: `startTime.getTime().toString()`?
* If test is running for 5 hours, how much it will be it second? Can we show it more flexible?

Reply:
- BlazeMeter test # 5021335, masterId # 15130441 running from Fri Dec 08 11:47:25 MSK 2017 - for 10 seconds
- we can show it in minutes. 1 min, 1.5 min, 2 min and so on.

Reply:
"Fri Dec 08 12:15:58 MSK 2017 - for 0.2 minutes"